src/PowerSystems2PRAS.jl

```diff
@@ -657,6 +657,161 @@ function process_genstorage(
     )
 end
 
+"""
+    $(TYPEDSIGNATURES)
+
+Apply LinePRAS to create PRAS matrices for lines. Views should be passed in.
+"""
+function assign_to_line_matrices!(
+    ::LinePRAS,
+    line::PSY.Branch,
+    s2p_meta::S2P_metadata,
+    forward_cap,
+    backward_cap,
+)
+    fill!(forward_cap, floor(Int, line_rating(line).forward_capacity))
+    fill!(backward_cap, floor(Int, line_rating(line).backward_capacity))
+end
+
+"""
+    $(TYPEDSIGNATURES)
+
+Create PRAS from sorted lines and formulations.
+"""
+function process_lines(
+    sorted_lines::Vector{PSY.Branch},
+    s2p_meta::S2P_metadata,
+    lines_to_formulation::Dict{PSY.Device, LinePRAS},
+)
+    num_lines = length(sorted_lines)
+    if num_lines == 0
+        line_names = String[]
+    else
+        line_names = PSY.get_name.(sorted_lines)
+    end
+    # Lines
+    line_cats = line_type.(sorted_lines)
+
+    line_forward_cap = Matrix{Int64}(undef, num_lines, s2p_meta.N)
+    line_backward_cap = Matrix{Int64}(undef, num_lines, s2p_meta.N)
+    line_λ = Matrix{Float64}(undef, num_lines, s2p_meta.N) # Not currently available/ defined in PowerSystems
+    line_μ = Matrix{Float64}(undef, num_lines, s2p_meta.N) # Not currently available/ defined in PowerSystems
+
+    for (i, line) in enumerate(sorted_lines)
+        assign_to_line_matrices!(
+            lines_to_formulation[line],
```

Comment:
I'm not sure why we need to pass _formulation_ to _assign_to_lime_matrices!_ function

Reply:
The main reason is so that if we ever extend our support to additional lines, then we will be able to look at the formulation metadata.

Reply:
This makes sense as well.